.golangci.yml

```diff
@@ -3,30 +3,31 @@ linters-settings:
     locale: US
 
 linters:
-  # please, do not use `enable-all`: it's deprecated and will be removed soon.
-  # inverted configuration with `enable-all` and `disable` is not scalable during updates of golangci-lint
-  # disable-all: true
   enable:
     - golint
     - misspell
-
-  # don't enable:
-  # - gochecknoglobals
-  # - gocognit
-  # - godox
-  # - maligned
-  # - prealloc
+    - gocyclo
+    - bodyclose
+    - unconvert
```

Comment:
Detected unecessary conversions, like:
```
hello := uint64(20)
...
foo := unit64(hello)
```

Cases fixed are due to changing types over time, and making conversions unnecessary.